python/pwsat2_submitter.py

```diff
@@ -62,8 +62,9 @@ def __init__(self, credentials_file, initialTimestamp):
         dtformat = '%Y-%m-%d %H:%M:%S'
         self.initialTimestamp = (
             datetime.datetime.strptime(initialTimestamp, dtformat)
+            .replace(tzinfo=datetime.timezone.utc)
```

Comment:
I have just added the following test case to my script:
```python
    t3 = parse_timestamp('2025-10-02 20:50:03')
    assert str(t3) == '2025-10-02 20:50:03+00:00'
```
It works fine for all Python >= 3.8. Note that the timestamp I have used matches the format expected by `dtformat`. So we could use `parse_timestamp()` rather than `strptime()`  and be more general in the timestamp formats that we accept as `initialTimestamp`.